File: back-end/src/main/java/com/example/back_end/dtos/response/SolicitacaoFuncionarioResumoDTO.java
package com.example.back_end.dtos.response;

import com.example.back_end.enums.EstadoSolicitacao;

public record  SolicitacaoFuncionarioResumoDTO (
    Long id,
    String createdAt,
    String clienteNome,
    String descricaoProduto,
    EstadoSolicitacao estado
) {}

File: front-end/src/app/shared/dtos/solicitacao-funcionario-resumo.dto.ts
import { EstadoSolicitacao } from "../models/solicitacao.model";

export interface FuncionarioSolicitacaoResumoDTO {
  id: number;
  createdAt: string;
  clienteNome: string;
  descricaoProduto: string;
  estado: EstadoSolicitacao | string;
}